news.html

```diff
@@ -105,6 +117,15 @@ <h3 class="icon fa-calendar"> Feb 11, 2005</h3>
 
 								<!-- Content -->
 									<section id="content">
+										<h3>Mobile Celestia release</h3>
+                                        Version 1.0.0 of Celestia is now available for iOS and Android - see the <a href="download.html">download page</a>. The program was ported to mobile devices by Levin Li.
+										<hr />
+										<h3>New owner of the Celestia website</h3>
+                                        Alexandr Kuznetsov (Alexell) has left the project and transferred ownership rights to Hleb Valoshka (onetwothree), who has created a new server that hosts the Celestia Website and the Forum. For more info, see the <a href="/forum/viewtopic.php?f=17&t=20375" target="_blank">forum topic</a>.
+										<hr />
+										<h3>A new Celestia fork is debated</h3>
```

Comment:
Shouldn't "debated" be "created"?

Reply:
As far as I know, Hleb has been developing his repository since the beginning of 2018, almost a year before the announcement on the forum.
But we can choose a better word instead of "debated"

Reply:
So "announced"?

Reply:
Ok